packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -41,21 +41,28 @@ function composeObjectExpressionPropertyIntoString(
   return composedProperty;
 }
 
-export function getLoadArgument(node: TSESTree.MemberExpression): string {
+export function getLoadArgument(
+  node: TSESTree.MemberExpression
+): string | undefined {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
-    if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
-      return node.parent.arguments[0].value as string;
-    } else if (
-      node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.ObjectExpression
-    ) {
-      return composeObjectExpressionPropertyIntoString(
-        node.parent.arguments[0]
-      );
+    if (node.parent.arguments.length === 0) {
+      return undefined;
+    } else {
+      if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
```

Comment:
We should be able to flatten the nested conditional so it's less complicated to follow.